Author: Pino Toscano <pino@debian.org>
Description: Disable the warnings about not being able to dlopen ICU libraries.
 Most of the applications don't need ICU-based functions for locale-aware
 string upper- and lower-casing and comparison. If not present, Qt will use the
 own ways (i.e. those used also in Qt < 4.8).
Last-Update: 2012-05-21
Forwarded: not-needed

--- a/src/corelib/tools/qlocale_icu.cpp
+++ b/src/corelib/tools/qlocale_icu.cpp
@@ -83,7 +83,9 @@ bool qt_initIcu(const QString &localeStr
         // resolve libicui18n
         QLibrary lib(QLatin1String("icui18n"), QLatin1String(U_ICU_VERSION_SHORT));
         if (!lib.load()) {
+#if 0
             qWarning() << "Unable to load library icui18n" << lib.errorString();
+#endif
             status = ErrorLoading;
             return false;
         }
@@ -112,7 +114,9 @@ bool qt_initIcu(const QString &localeStr
         // resolve libicuuc
         QLibrary ucLib(QLatin1String("icuuc"), QLatin1String(U_ICU_VERSION_SHORT));
         if (!ucLib.load()) {
+#if 0
             qWarning() << "Unable to load library icuuc" << ucLib.errorString();
+#endif
             status = ErrorLoading;
             return false;
         }
